rust/libnewsboat/src/utils.rs

```diff
@@ -311,6 +312,26 @@ pub fn is_valid_podcast_type(mimetype: &str) -> bool {
     matches || found
 }
 
+pub fn get_auth_method(method: &str) -> u64 {
+    match method {
+        "basic" => curl_sys::CURLAUTH_BASIC,
+        "digest" => curl_sys::CURLAUTH_DIGEST,
+        "digest_ie" => curl_sys::CURLAUTH_DIGEST_IE,
+        "gssnegotiate" => curl_sys::CURLAUTH_GSSNEGOTIATE,
+        "ntlm" => curl_sys::CURLAUTH_NTLM,
+        "anysafe" => curl_sys::CURLAUTH_ANYSAFE,
+        "any" | "" => curl_sys::CURLAUTH_ANY,
+        _ => {
+            log!(
+                Level::UserError,
+                "utils::get_auth_method: you configured an invalid proxy authenticate method: {}",
```

Comment:
Sorry, this still says "authenticate" rather than "authentication". I feel like I'm nitpicking, but c'mon, we had a proper message, why not just copy it.

Reply:
Sorry my bad, I rarely copy paste when the sentence is short.